### DISCUSSION TITLE
Site view stats

### DISCUSSION THREAD

**Discussion by michalceb** [netbox-community/netbox#8972] [Q&A] [answered]
When we have upgraded our instance of Netbox to version 3.1.9, I have noticed that the usual "Stats" section in Site View (the same one that we see in Tenant View):
![image](https://user-images.githubusercontent.com/67379333/160163051-ecafdb05-fc89-468c-ab4e-e646436136a7.png)

has changed to a "Related objects" section:
![image](https://user-images.githubusercontent.com/67379333/160163150-dcdb5c45-d3d2-413a-ab10-a413bdcee60d.png)

I had a few complaints from our users that it is less functional and just wanted to check if that was an intended change or a bug.
I can see the same change in the Demo instance, so I don't think this is a bug just in our Netbox.

**Comment by jeremystretch** [comment, marked as answer]
It's very much intentional. The change was made to improve readability of the labels and to better conform to the rest of the UI.
